### DISCUSSION TITLE
FindIndex for ImmutableArray

### DISCUSSION THREAD

**Discussion by acaly** [dotnet/runtime#45340] [Q&A] [answered]
Is there an equivalent to the ```Array.FindIndex``` for immutable arrays, which accepts a ```Predicate```? Or is there a reason why this method is not provided?

**Comment by danmoseley** [comment]
@eiriktsarpalis

**Comment by eiriktsarpalis** [comment, marked as answer]
There doesn't seem to be one. I can't think of a reason why we couldn't add this. Would you be interested in filing an [API proposal](https://github.com/dotnet/runtime/blob/master/docs/project/api-review-process.md)?

**Reply by acaly** [reply]
Thank you for the reply!

Yes, I can write one for that. I can see some other members missing, but I believe it's better to discuss the scope in that separate issue.

**Reply by acaly** [reply]
@eiriktsarpalis Please see #45366.
